src/core/layout/navbar/navigation/cameras/hooks/use-navigation.jsx

```diff
@@ -0,0 +1,10 @@
+import { useCallback } from 'react'
+import { useHistory } from 'react-router-dom'
+
+export const useNavigation = () => {
```

Comment:
Since there's no jsx syntax here, let's rename this file to `use-navigation.js`

Reply:
Let's also combine all these `useNavigation` hooks into one that receives the path as its parameter.

Reply:
Done